{% extends "layout.html" %}
  
{% block title %}Results{% endblock %}
{% block lead %}You can describe the results of your project and your future plans here.{% endblock %}

{% block page_content %}

<header class="hero">
    <h1 style="margin-bottom: 30vh;">Result</h1>
    <div class="scroll-indicator">↓</div>
  </header>
  
  <main class="container main-content">
    <aside class="table-of-contents" style="width: 15vw;">
        <h2>Table of Contents</h2>
        <ul>
            <li><a href="#section-1"  style="margin-top: 0vh;">Gram Stain</a></li>
            <li><a href="#section-2">Hot Spore Stain</a></li>
            <li><a href="#section-3">Indole Test</a></li>
            <li><a href="#section-4">MR-VP Test</a></li>
            <li><a href="#section-5">Oil Emulsification Test</a></li>
            <li><a href="#section-6">Growth Curve</a></li>
            <li><a href="#section-7">Iturin A Extraction</a></li>
            <li><a href="#section-8">HPLC</a></li>
            <li><a href="#section-9">Antifungal Assay</a></li>
            <li><a href="#section-10">Fragment Amplification</a></li>
            <li><a href="#section-11">Homologous Recombination</a></li>
        </ul>
    </aside>
  
    <div class="content-panel">
        <div class="research-content">
            <style>
                .result_titles{
                    font-size: 3vh !important;
                }
            </style>

            <section id="section-1">
                    <h2 class="result_titles">1. Gram Staining</h2>
                    <p>
                        Bacillus subtilis ATCC 13952 cells were gram stained and violet colour cells with rod-like morphology were observed.  
                        The bacteria were confirmed to be gram positive Bacillus subtilis.
                        <center><figure>
                            <img style="width:20vw;height: auto;" src="https://static.igem.wiki/teams/5141/results/gram-staining-100x.jpeg" alt="">
                            <figcaption>Fig: Gram-staining-100x</figcaption>
                        </figure></center>
                    </p>
            </section>

            <section id="section-2">
                    <h2 class="result_titles">2. Hot Spore Staining</h2>
                    <p>
                        As Bacillus subtilis is a spore forming bacteria, the bacteria were stained to visualize the endospores. An overgrown culture was used for this stain as bacteria tend to form spores under stress such as lack of nutrients. Endospore formation was observed.  
                        Faint green endospores were visualized which proved the spore forming nature of our bacteria.  
                    </p>
                </section>

            <section id="section-3">
                    <h2 class="result_titles">3. Indole Test</h2>
                    <p>
                        Spot indole test was conducted and a yellow colour was observed.  
                        A negative indole test confirmed that the organism cannot breakdown tryptophan to produce indole thus categorizing it with most other Bacillus species.  
                        <center><figure>
                            <img style="width:30vw;height: auto;" src="https://static.igem.wiki/teams/5141/results/indole-test.png" alt="">
                            <figcaption>Fig: Indole-test</figcaption>
                        </figure></center>
                    </p>
                </section>

            <section id="section-4">
                    <h2 class="result_titles">4. MR-VP Test</h2>
                    <p>
                        <strong>Methyl Red test:</strong> Negative  
                        <br>The test showed no colour change  
                        <br><strong>Voges-Proskauer:</strong> Positive  
                        <br>The test showed yellow colour with no colour change.  
                        The bacteria does not produce stable acidic end products from glucose fermentation and uses butylene glycol pathway to produce neutral end products like actoin. This confirms that the bacteria is Bacillus subtilis.  
                        <center><figure>
                            <img style="width:30vw;height: auto;" src="https://static.igem.wiki/teams/5141/results/mr-test.jpeg" alt="">
                            <figcaption>Fig: Methyl Red(MR) Test</figcaption>
                        </figure></center>
                
                        <center><figure>
                            <img style="width:30vw;height: auto;" src="https://static.igem.wiki/teams/5141/results/vp-test.jpeg" alt="">
                            <figcaption>Fig: Voges-Proskauer(VP) Test</figcaption>
                        </figure></center>
                    </p>
                </section>

            <section id="section-5">
                    <h2 class="result_titles">5. Oil Emulsification Test</h2>
                    <p>
                        Upon addition of 100 µl of broth onto the plate containing petrol, the Biosurfactant activity of the Iturin A in the broth created a zone of clearance was created due to the interaction.  
                        100 µl of broth supernatant, Extracted Iturin A and pure Iturin A was added to each test tube containing water and petrol and were shaken for 5 minutes and observed for emulsification.  
                        All these tubes showed emulsification with highest activity from the Pure Iturin A sample and least in the test tube containing the broth supernatant.  
                        <center><figure>
                            <img style="width:30vw;height: auto;" src="https://static.igem.wiki/teams/5141/results/emulsification-of-oil.png" alt="">
                            <figcaption>Fig: Oil Emulsification Test</figcaption>
                        </figure></center>                
                    </p>
                </section>

            <section id="section-6">
                    <h2 class="result_titles">6. Growth Curve</h2>
                    <p>
                        24-hour growth curve was first experimented with an inoculation volume of 100 µl of broth. However, the growth curve observed was not very distinct with respect to the various stages of growth. Diauxic growth curve was observed in the 24-hour growth as well.  
                        Upon consultation with Dr. Dr Sabari Sankar Thirupathy, we obtained the optimized protocols to add the 0.7 OD culture in a 1:1000 dilution factor and run the growth curve for 6 hours. However, there was hardly any growth in the medium for the first 4 hours and hence the graph plotted was inconclusive.  
                        Our 4th trial based on 1:500 dilution and a 12-hour long growth, gave a distinct and accurate graph.  
                        The bacteria showed Diauxic growth initially due to the presence of multiple sugars in the LB medium. Upon exhaustion of these sugars, we observed an exponential phase spanning over 3 hours and then proceeded into stationary phase.  
                        Bacillus subtilis growth aligns with the various growth curves as mentioned in many literatures.  
                        <center><figure>
                            <img style="width:30vw;height: auto;" src="https://static.igem.wiki/teams/5141/results/12-hours-growth-curve.png" alt="">
                            <figcaption>Fig: 12-Hours-Growth-Curve</figcaption>
                        </figure></center>
                    </p>
                </section>

            <section id="section-7">
                    <h2 class="result_titles">7. Iturin A Extraction from LB (Luria-Bertani) media and PDB (Potato Dextrose Agar) media</h2>
                    <p>
                        Iturin A was extracted from two sets of growth media – LB and PDB giving us a comparative analysis of which medium is better for extraction; this was analyzed using HPLC.  
                        The total volume of extracted sample was 8ml in methanol which was reduced to 4 ml upon filtration.  
                        <center><figure>
                            <img style="width:30vw;height: auto;" src="https://static.igem.wiki/teams/5141/results/centrifuge-pellet.png" alt="">
                            <figcaption>Fig: Iturin A Extraction, Centrifuged pellets</figcaption>
                        </figure></center>
                    </p>
                </section>

            <section id="section-8">
                        <h2 class="result_titles">8. HPLC (High pressure liquid chromatography) of Extracted Iturin A and pure Iturin A</h2>
                        <p>
                            Standard of pure Iturin A sample was first run at variable concentrations ranging from 100µg/ml to 1.5625µg/ml and multiple peaks of Iturin A isomers were obtained. 7 isomer peaks were observed.  
                            A standard calibration curve was made by calculating the cumulative area of all the peaks at various concentrations and were observed to be linear.  
                            LB gave sharper peaks due to lesser media complexity in terms of the various elements present in the media like beef extract, peptone etc. as compared to PDB which is highly complex in its composition leading to unnecessary peaks or impurities.  
                            Despite the lesser sharper peaks of the Iturin A extracted in PDB, the quantity of extraction was greater than in LB as more area was calculated in PDB as compared to LB.  
                            Using the calibration curve, the concentration of the crude extracted Iturin A was estimated. The extract's concentration was low due to high dilution in methanol during the extraction and filtration process. Giving a total concentration of 1.472 µg/ml in LB and 3.344 µg/ml in PDB, respectively.  
                            
                            <center><figure>
                                <img style="width:30vw;height: auto;" src="https://static.igem.wiki/teams/5141/results/standard-alingment.png" alt="">
                                <figcaption>Fig: Standard Alignment</figcaption>
                            </figure></center>
                    
                            <center><figure>
                                <img style="width:30vw;height: auto;" src="https://static.igem.wiki/teams/5141/results/standard-vs-lb-extracted-iturin-a.png" alt="">
                                <figcaption>Fig: Standard vs LB-extracted-iturin-A</figcaption>
                            </figure></center>
                    
                            <center><figure>
                                <img style="width:30vw;height: auto;" src="https://static.igem.wiki/teams/5141/results/standard-vs-pdb-extracted-iturin-a.png" alt="">
                                <figcaption>Fig: Standard vs PDB-extracted-iturin-A</figcaption>
                            </figure></center>

                            In the above-mentioned images the Black color graph is the standard while the purple graph is the test graph.
                        </p>
                    </section>

            <section id="section-9">
                        <h2 class="result_titles">9. Antifungal Assay</h2>
                        <p>
                            Unfortunately, there was no zone of clearance observed due to the culture being too overgrown. The experiment will be repeated on a freshly cultured fungal lawn to test the antifungal activity of Iturin A and the extracted Iturin A samples.
                        </p>
                    </section>

            <section id="section-10">
                        <h2 class="result_titles">10. Fragment Amplification</h2>
                        <p>
                            The fragments were amplified, and our results were as follows:  
                            <center><figure>
                                <img style="width:30vw;height: auto;" src="https://static.igem.wiki/teams/5141/results/amplified-pbaca-fragment.webp" alt="">
                                <figcaption>Fig: Amplified PbacA fragment</figcaption>
                            </figure></center>
                    
                            <center><figure>
                                <img style="width:30vw;height: auto;" src="https://static.igem.wiki/teams/5141/results/amplified-p43-fragment.webp" alt="">
                                <figcaption>Fig: Amplified P43 fragment</figcaption>
                            </figure></center>
                            The amplification results demonstrate that the PbacA fragment was successfully amplified, while the P43 fragment produced only a faint band. This can be attributed to the suboptimal performance of the primers designed for the P43 fragment, which was affected by the following factors:  
                            <ul>
                                <li>Many of the primer combinations formed hairpin structures and primer dimers.</li>
                                <li>The ΔG (Gibbs free energy) values of the primers were slightly lower than the optimal range.</li>
                            </ul>
                            The most favourable primer combination was selected to move our project forward. However, it remained suboptimal and resulted in weak amplification.  
                            The primers used are specified in the table below: 
                            <center><figure>
                                <img style="width:50vw;height: auto;" src="https://static.igem.wiki/teams/5141/results/primer-data-sheet.webp" alt="">
                                <figcaption>Fig: Primer Data Sheet</figcaption>
                            </figure></center> 
                        </p>
                    </section>

            <section id="section-11">
                        <h2 class="result_titles">11. Homologous Recombination</h2>
                        <p>
                            Homologous recombination was performed using protocols obtained from Dr. Sabari Sankar Thirupathy, a professor at IISER Trivandrum. These protocols were optimized for our lab, and all precautionary measures were taken before experimenting to ensure the best results.  
                            Our selectable marker was GFP, and this was visualized in the transformed colonies at an emission wavelength of 509 nm using a Gel dock.  
                            The homologous recombination done with our PbacA fragment yielded successful results with a clear demarcation of green fluorescent colonies, whereas P43 did not yield successful results.  
                            The results are as follows:  
                            <center><figure>
                                <img style="width:40vw;height: auto;" src="https://static.igem.wiki/teams/5141/results/hr-gfp-pbaca-result.webp" alt="">
                                <figcaption>Fig: Homologous Recombination GFP result - PbacA</figcaption>
                            </figure></center>

                            <center><figure>
                                <img style="width:40vw;height: auto;" src="https://static.igem.wiki/teams/5141/results/hr-gfp-p43-result.webp" alt="">
                                <figcaption>Fig: Homologous Recombination GFP result - P43</figcaption>
                            </figure></center>
                        </p>
                        <p>
                            What we learnt from the homologous recombination experiments we conducted are as follows:  
                            <ul>
                                <li>The negative HR result for our P43 fragment was due to its low amplification even after several tries.</li>
                                <li>Potassium L-glutamate monohydrate is a component that increases the efficiency of homologous recombination (HR). The amount we added to the HR media (1X MC) was insufficient, and by using a higher concentration, we could have potentially increased the probability of HR occurring and observed more fluorescent colonies.</li>
                            </ul>
                            The above learnings will be implemented in future HR experiments that will be conducted.
                        </p>
                    </section>
        </div>
    </div>
                
  </main> 

{% endblock %}
